draft-ietf-dnsop-svcb-https.md

```diff
@@ -227,7 +227,7 @@ for the case of "https" origins as described in {{https}}.
 SVCB RRs are extensible by a list of SvcParams, which are pairs consisting of a
 SvcParamKey and a SvcParamValue. Each SvcParamKey has a presentation name and a
 registered number. Values are in a format specific to the SvcParamKey. Their
-definition should specify both their presentation format and wire encoding
+definition must specify both their presentation format and wire encoding
```

Comment:
Is a different lower-case keyword an improvement here?  Given that the rest of this section is descriptive rather than normative, perhaps "SvcParam definitions specify...."

Reply:
Done